task-launcher/src/tasks/shared/helpers/fetchAndParseCorpus.js

```diff
@@ -156,7 +144,7 @@ export const fetchAndParseCorpus = async (config) => {
 
   const csvTransformed = {
     practice: sequentialPractice ? practiceData : _shuffle(practiceData),
-    stimulus: sequentialStimulus ? stimulusData : _shuffle(stimulusData),
```

Comment:
Shuffling here will mix the instructions, practice, and stimulus

Reply:
Could you remove the practice property here, and also remove practiceData not that we don't use it.